conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -44,6 +44,9 @@ conda update --yes --all
 conda install --yes conda-build==1.18.2
 conda info
 
+# Installs anything from a \`yum_requirements.txt\` file in a recipe with \`yum\`.
+test -f "recipe_root/yum_requirements.txt" && (cat "recipe_root/yum_requirements.txt" | xargs -r yum install -y) || true
```

Comment:
@jakirkham - would you mind pulling this into line with the version at staged-recipes: https://github.com/conda-forge/staged-recipes/blob/66acd84f415404fe538843983e542e76a620cca4/scripts/run_docker_build.sh#L45-L48.